package.json

```
@@ -1,101 +1,88 @@
+        "build:cem": "wireit",
+        "build:icons": "wireit",
+        "build:react": "wireit",
+        "build:tests": "wireit",
         "build:ts": "wireit",
-        "build:ts:watch": "wireit",
         "build:types": "wireit",
-        "build:watch": "wireit",
-        "changeset-snapshot-publish": "yarn prepublishOnly && yarn changeset version --snapshot snapshot && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag --tag snapshot",
-        "changeset-publish": "yarn prepublishOnly && yarn changeset version && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag && yarn push-to-remote && yarn create-git-tag && yarn postpublish",
-        "update-version": "node ./tasks/update-version.js",
         "chromatic": "chromatic --build-script-name storybook:build # note that --project-token must be set in your env variables",
-        "create-git-tag": "node --no-warnings tasks/create-git-tag.js",
-        "custom-element-json": "node tasks/custom-element-json.js",
-        "docs:analyze": "cem analyze --globs \"packages/**/*.ts\" --exclude \"**/*.d.ts\" --exclude \"**/stories/**\" --exclude \"**/icons/**\" --exclude \"**/elements/**\" --outdir projects/documentation --litelement",
         "docs:build": "yarn workspace documentation build",
-        "docs:ci": "yarn docs:analyze && run-p docs:production storybook:build && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
-        "docs:preview": "yarn docs:analyze && run-p docs:build storybook:build && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
+        "docs:cem": "wireit",
+        "docs:ci": "yarn docs:cem && run-p docs:production storybook:build:docs && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
+        "docs:preview": "yarn docs:cem && run-p docs:build storybook:build:docs && cp projects/documentation/custom-elements.json projects/documentation/dist/storybook",
         "docs:production": "yarn workspace documentation build:production",
         "docs:review": "alex packages/**/*.md",
         "docs:start": "yarn workspace documentation serve --watch",
-        "find": "test -f custom-elements.json",
-        "format:css": "yarn lint:css --fix && pretty-quick --pattern \"{packages,tools}/**/*.css\"",
-        "gen-react-wrapper": "node ./tasks/gen-react-wrapper.js",
-        "icons": "wireit",
-        "icons:ui": "wireit",
-        "icons:workflow": "wireit",
         "lint": "run-p lint:js lint:docs lint:ts lint:css lint:packagejson",
         "lint:css": "stylelint \"packages/**/*.css\" \"tools/**/*.css\"",
         "lint:docs": "eslint -f pretty \"projects/documentation/**/*.ts\"",
-        "lint:js": "pretty-quick --pattern \"tasks/**/*.js\" && pretty-quick --pattern \"scripts/**/*.js\"",
+        "lint:js": "pretty-quick --pattern \"scripts/**/*.js\"",
```

```suggestion
        "lint:scripts": "pretty-quick --pattern \"scripts/**/*.js\"",
```

nit: more explicit since its not all javascript files

---

I think it _could_ be all raw JS files (not the generated ones from typescript though). We just haven't expanded the pattern yet to cover those use-cases.